<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addretmaterial</class>
 <widget class="QWidget" name="addretmaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetData">
     <column>
      <property name="text">
       <string>原料名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>原料编码</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetEditData">
     <column>
      <property name="text">
       <string>原料名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>原料编码</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBoxAllSelect">
       <property name="text">
        <string>全选</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res.qrc">
         <normaloff>:/image/image/全选.png</normaloff>:/image/image/全选.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string> 保存</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res.qrc">
         <normaloff>:/image/image/save.png</normaloff>:/image/image/save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDel">
       <property name="text">
        <string> 删除</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res.qrc">
         <normaloff>:/image/image/删 除.png</normaloff>:/image/image/删 除.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string> 取消</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res.qrc">
         <normaloff>:/image/image/cancel.png</normaloff>:/image/image/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../res.qrc"/>
 </resources>
 <connections/>
</ui>
